neqo-transport/src/recovery.rs

```diff
@@ -318,6 +318,19 @@ impl LossRecovery {
         self.cc.ssthresh()
     }
 
+    #[cfg(test)]
+    pub fn latest_rtt(&self) -> Duration {
+        self.rtt_vals.latest_rtt
+    }
+
+    pub fn smoothed_rtt(&self) -> Option<Duration> {
+        self.rtt_vals.smoothed_rtt
+    }
+
+    pub fn set_latest_rtt(&mut self, value: Duration) {
```

Comment:
I would change the number of this function name to `set_initial_rtt()`.  Then add an assertion that you have no prior estimates.

```suggestion
    pub fn set_initial_rtt(&mut self, value: Duration) {
        debug_assert!(self.smoothed_rtt.is_none());
```